### PR TITLE
Setting up ionic

### PR DESCRIPTION
**Pull request by eripheebs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,25 @@ Playfest
 To set up:
 1. fork the repo
 2. npm install
-3. if you don't already have these packages installed globally:
-  npm install -g mocha
-
+3. (if you don't already have these packages installed globally)
+  npm install -g .. blah blah..
+4. cd frontend
+5. npm install
```
Comment (robota-x) [inline README.md]: 5.+ npm install && bower install (required for Ionic dependencies)

File: .gitignore [hunk 2]
```diff
@@ -1 +1,4 @@
 node_modules
+frontend/platforms
+frontend/plugins
+frontend/www/lib/js
```
Comment (robota-x) [inline .gitignore:4]: change to frontend/www/lib to include all the packets installed with bower?

Comment (eripheebs) [inline .gitignore:4]: lib includes some stuff that isnt generated.

**Comment by robota-x** [comment]
:shipit:

**Comment by JojSh** [comment]
:shipit:

**Comment by jazzygasper** [comment]
:shipit:

**Pull request merged**